modules/ai/preferences.php

```diff
@@ -0,0 +1,114 @@
+<?php
+namespace Elementor\Modules\Ai;
+
+use Elementor\User;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Preferences {
+
+	const ENABLE_AI = 'elementor_enable_ai';
+
+	/**
+	 * Register actions and hooks.
+	 *
+	 * @return void
+	 */
+	public function register() {
+		add_action( 'profile_personal_options', function ( \WP_User $user ) {
+			$this->add_personal_options_settings( $user );
+		} );
+
+		add_action( 'personal_options_update', function ( $user_id ) {
+			$this->update_personal_options_settings( $user_id );
+		} );
+	}
+
+
+
+	/**
+	 * Determine if AI features are enabled for a user.
+	 *
+	 * @param int $user_id - User ID.
+	 *
+	 * @return bool
+	 */
+	public static function is_ai_enabled( $user_id ) {
+		return ! ! User::get_user_option_with_default( static::ENABLE_AI, $user_id, true );
```

Comment:
```suggestion
		return (bool) User::get_user_option_with_default( static::ENABLE_AI, $user_id, true );
```
Maybe? More common and clear.